server/xpub-model/migrations/1552580166-app-audit.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TYPE auditaction RENAME TO auditaction_old;
```

Comment:
You probably committed this by mistake - this is to do with the AuditLog changes